langlib/lang.decimal/src/main/ballerina/decimal.bal

```diff
@@ -110,6 +176,14 @@ public isolated function ceiling(decimal x) returns decimal = @java:Method {
 # - a FloatingPointTypeSuffix is not allowed
 # This is the inverse of function ``value:toString`` applied to an `decimal`.
 #
+# ```ballerina
+# decimal:fromString("0.2453") ⇒ 0.2453
+# 
+# decimal:fromString("-10") ⇒ -10
+# 
+# decimal:fromString("") ⇒ error
```

Comment:
Shall we use an actual erroneous string instead of empty str?

Reply:
can you share the review comment

Reply:
https://github.com/ballerina-platform/ballerina-lang/pull/38905#discussion_r1049245698

Reply:
I was thinking
```ballerina
# decimal:fromString("123d") ⇒ error
```
@MaryamZi wdyt?

Reply:
+1, as long as it is not an obviously non-parseable string.

Reply:
Ok, will update

Reply:
Updated